BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/SplashActivity.java

```diff
@@ -18,10 +18,9 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         SharedPreferences sharedPref = getSharedPreferences(getString(R.string.preference_file_key), Context.MODE_PRIVATE);
         String token = sharedPref.getString(getString(R.string.preference_login_token), "");
-        boolean isGuest = sharedPref.getBoolean(getString(R.string.preference_logged_in_as_guest), false);
-        Log.d(TAG, token + " " + isGuest);
+        Log.d(TAG, token);
```

Comment:
FYI this was just there for debugging purposes, you can (and probably should) remove it. My bad :)